### PR TITLE
fix(web): domain email delegation

### PR DESCRIPTION
**Pull request by frenautvh**
See : DTRSD-2653

### CODE REVIEW

File: packages/manager/apps/web/client/app/email-domain/delegate/email-domain-delegate.controller.js [hunk 1]
```diff
@@ -125,14 +125,16 @@ angular.module('App').controller(
       return this.$q
         .all({
           email: this.WucEmails.getDelegatedEmail(item),
-          usage: this.WucEmails.getEmailDelegatedUsage(item),
+          usage: this.WucEmails.getEmailDelegatedUsage(item).catch(() => null),
```
Comment (marie-j) [inline packages/manager/apps/web/client/app/email-domain/delegate/email-domain-delegate.controller.js]: This is the wanted behaviour for all API errors ? Even 500 for instance ?

Comment (FredericEspiau) [reply packages/manager/apps/web/client/app/email-domain/delegate/email-domain-delegate.controller.js]: @frenautvh yes ? maybe ? I don't know ? Can you repeat the question ?

Comment (frenautvh) [reply packages/manager/apps/web/client/app/email-domain/delegate/email-domain-delegate.controller.js]: hello @marie-j @FredericEspiau, my idea was that if the call to usage fail then we simply ignore the value and display a "-" in the datagrid. Do you want me to add an error message? if so, what kind of message : a single one if one or more usage call fail or for example one red pill in the datagrid for each failed call?

Comment (antleblanc) [reply packages/manager/apps/web/client/app/email-domain/delegate/email-domain-delegate.controller.js]: @frenautvh @marie-j @FredericEspiau here we are addressing a hotfix so I'm agree with the first suggestion that tends to display the `-` character inside the datagrid.

Comment (FredericEspiau) [reply packages/manager/apps/web/client/app/email-domain/delegate/email-domain-delegate.controller.js]: @antleblanc we don't want the remedy to be worse than the poison 😉 

Also, if it's a hotfix, just merge it already, a hotfix can't last for 21 days 😄

**Pull request merged**